static/app/views/eventsV2/metricsBaselineContainer.tsx

```diff
@@ -75,7 +77,11 @@ export function MetricsBaselineContainer({
     ? getUtcToLocalDateObject(pageFilters.datetime.end)
     : null;
 
-  const showBaseline = location.query.baseline === '0' ? false : true;
+  const showBaseline =
+    (location.query.baseline ?? localStorage.getItem(PROCESSED_BASELINE_TOGGLE_KEY)) ===
+    '0'
+      ? false
+      : true;
```

Comment:
```suggestion
  const showBaseline =
    (location.query.baseline ?? localStorage.getItem(PROCESSED_BASELINE_TOGGLE_KEY)) === '1'
```

Wouldn't this also work since `===` returns a boolean

Reply:
omg haha yes

Reply:
actually i think we want to default it to true

Reply:
like if there isn't a value in the location query or in local storage, we'd want to still have a default of true so it's more discoverable

Reply:
Oh, right, bah lol